.github/workflows/actions.yml

```
@@ -0,0 +1,95 @@
+# https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions
+# https://packaging.python.org/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/
+#
+# Needed GitHub Repository Secret:
+#   PYPI_PASSWORD - PyPI API token allowing write, release access to PyPI.
+#     Limit scope to just project.
+#     (See package.python.org example link above.)
+
+name: On push
```

I think this name will be displayed in a lot of places, how about `Continuous Integration` or something?

```suggestion
name: Continuous Integration
```